src/java.base/share/classes/java/net/Socket.java

```diff
@@ -378,26 +378,19 @@ public Socket(InetAddress address, int port, InetAddress localAddr,
      * In other words, it is equivalent to specifying an address of the
      * loopback interface. </p>
      * <p>
-     * If the stream argument is {@code true}, this creates a
-     * stream socket. If the stream argument is {@code false}, it
-     * creates a datagram socket.
-     * <p>
```

Comment:
One final though on the Socket changes is that maybe we should include an API note to say that the stream parameter provided a way in early JDK releases to create a Socket that used a datagram socket, this feature no longer exists.

Reply:
Done